main/chains/feeTransformers/index.js

```diff
@@ -0,0 +1,16 @@
+const { gweiToWei } = require('../../../resources/utils')
+const { intToHex } = require('@ethereumjs/util')
+
+export default {
```

Comment:
as mentioned on standup today, let's move this logic into an encapsulated gas calculator thats specific to Polygon. maybe we can leverage the `chainConfig` function that we use to also configure the gas calculator as well as other things in the future?